packages/jobs/lib/runner/render.runner.ts

```diff
@@ -79,7 +80,9 @@ export class RenderRunner implements Runner {
                         { key: 'NANGO_TELEMETRY_SDK', value: process.env['NANGO_TELEMETRY_SDK'] || 'false' },
                         { key: 'DD_ENV', value: process.env['DD_ENV'] || '' },
                         { key: 'DD_SITE', value: process.env['DD_SITE'] || '' },
-                        { key: 'DD_TRACE_AGENT_URL', value: process.env['DD_TRACE_AGENT_URL'] || '' }
+                        { key: 'DD_TRACE_AGENT_URL', value: process.env['DD_TRACE_AGENT_URL'] || '' },
+                        { key: 'PROVIDERS_URL', value: getProvidersUrl() },
+                        { key: 'PROVIDERS_RELOAD_INTERVAL', value: envs.PROVIDERS_RELOAD_INTERVAL.toString() }
```

Comment:
Note: the env vars are only passed when the runner is created, which means we are gonna need to backfill those values for all existing runners. I think I have done it once in the past, not sure I pushed the script though. I remember the Render api around env vars was pretty awkward

Reply:
Wrote up a script here: https://github.com/NangoHQ/nango/pull/2984